CONTRIBUTING.md

```diff
@@ -38,3 +38,31 @@ owners](https://github.com/orgs/openSUSE/teams/owners) know!
 GitHub issues are the primary way for communicating about specific proposed
 changes to this project. If you have other problems please use one of the other
 [support channels])(http://openbuildservice.org/support/)
+
+# Rubocop
+We are currently in the process of adding rubocop rules to OBS. For that we
+frequently meet, decide on new rules to add and afterwards go through that list
+and fix those.
+
+Since we want to make sure that the number of merge conflicts stays as small as
+possible, we mark rubocop offenses with name tags (in .rubocop.yml). Developers are
+only supposed to work on a rubocop offense, if there is no name tag above theirs;-)
+
+If you want to take part of this please follow this process:
+
+* Make sure noone else is working on rubocop issues. (By Checking that your name
+  is on top of the .rubocop.yml file).
+
+* Pick one rubocop rule and enable it in .rubocop.yml. Make sure there are no
+  excludes for that offense in the .rubocop_todo.yml.
+
+* Run rubocop and fix reported offenses.
+
+* Remove the 'Enabled: true' from the .rubocop.yml. But keep the rule there. That way
```

Comment:
Na, I meant it that way. The 'rubocop --auto-gen-config' call would remove the 'Enabled: false' from the todo list ([like here](https://github.com/bgeuken/open-build-service/commit/b7a45d390fc11e8952b15b26143c9a877d859b9f#diff-81b31b1bff90e45a674c38b6d10d84b2L1133)). So we could remove the explicit enablement.

But it might be less error prone to keep the enablement until we got reid of the todo list. I'll update the PR